internal/venice-client-common/src/main/java/com/linkedin/venice/utils/AvroSchemaUtils.java

```diff
@@ -1,5 +1,7 @@
 package com.linkedin.venice.utils;
 
+import static org.apache.avro.Schema.Type.*;
```

Comment:
Nit:
can we get rid of wildcard import?

Reply:
Sure